jsk_fetch_robot/jsk_fetch_startup/README.md

```diff
@@ -213,6 +213,26 @@ Show logs
 tmuxinator log
 ```
 
+## Web UI
+
+There are some web UI deployed to Fetch.
+
+### Supervisor Web UI
+
+See [Supervisor section](#supervisor).
+
+### rwt_app_chooser
+
+See [Apps section](#apps).
+
+### webvideo
+
+http://webvideo.fetch1075.jsk.imi.i.u-tokyo.ac.jp/
```

Comment:
For security reasons, it would be preferable not to paste the URL directly.

Other than that, I think it is good.

Reply:
For automatic crawler, it is too late.
For manual attacker, it is not effective to hide URL to avoid attack because it can be easily guessed from the files in the repository.

Reply:
It is not an extreme issue of effectiveness/ineffectiveness. I don't think it's completely ineffective.

At least, please change the host name from fetch1075 to a fetch common name.

Reply:
it's never too late.
let's do good things.

Reply:
We need to discuss this issue more.

Reply:
そんなに難しい話ではなくて，気付けるところは直していきましょうでいいんじゃないかな．
全てかゼロかの話ではないし，今後の教育的観点からも替えておいたほうがいいんじゃないかな．
何を議論するところがあるのかわからないけど，メールアドレスをそのまま入れていることは避けるべきであって，そこを完全に避けたいという話はしていない．そういう態度をとって教育的に啓蒙することに価値がある．と考えるべきかな．
というかここで議論してもしかたないのでマージするかしないかをさっさと決めてほしい．
ここでちんたらしてたら北川のところでbranchをきって早く開発を回している意味がない．
原理的にはメールアドレスを消したほうがいいと思うけど，そうじゃないもう手遅れだと言っている人がいるのであれば，もうそれは一旦置いておいてマージすればいいんじゃないかな．
手遅れかそうでないかという議論は何一つとして進展を生み出さないので，手間でない限りは良いと思う方向に先に進みましょう．